odata-csdl/Appendix.md

```diff
@@ -152,6 +152,254 @@ https://www.ogc.org/standard/sfa/.
+- ᴘ IncludeInServiceDocument: Boolean (optional default: false))
+- ᴘ Name: String 
+
+##### <span id="key-element"/>Key Element>/span>
+
+  - ↑ [EntityType](#entitytype-element)
+  - ↓ [PropertyRef](#propertyref-element)
+
+##### <span id="member-element"/>Member Element>/span>
+
+  - ↑ [EnumType](#enumtype-element)
+  - ↓ [Annotation](#annotation-element)
+- ᴘ Name: String 
+- ᴘ Value: Int64 (optional)
+
+##### <span id="navigationproperty-element"/>NavigationProperty Element>/span>
+
+  - ↑ [EntityType](#entitytype-element), [ComplexType](#complextype-element)
+  - ↓ [ReferentialConstraint](#referentialconstraint-element), [OnDelete](#ondelete-element), [Annotation](#annotation-element)
+- ᴘ ContainsTarget: Boolean (optional)
+- ᴘ MaxLength: Int64 (optional)
+- ᴘ Name: String 
+- ᴘ Nullable: Boolean (optional)
+- ʀ Partner: [NavigationProperty](#navigationproperty-element) (optional)
+- ᴘ Precision: Int64 (optional)
+- ᴘ Scale: Scale (optional)
+- ᴘ SRID: Int64 (optional)
+- ʀ Type: [EntityType](#entitytype-element) 
+- ᴘ Unicode: Boolean (optional)
+
+##### <span id="navigationpropertybinding-element"/>NavigationPropertyBinding Element>/span>
+
+  - ↑ [EntitySet](#entityset-element), [Singleton](#singleton-element)
+- ᴚ Path 
+- ᴚ Target 
+
+##### <span id="ondelete-element"/>OnDelete Element>/span>
+
+  - ↑ [NavigationProperty](#navigationproperty-element)
+  - ↓ [Annotation](#annotation-element)
+- ᴘ Action: OnDeleteAction 
+
+##### <span id="parameter-element"/>Parameter Element>/span>
+
+  - ↑ [Action](#action-element), [Function](#function-element)
+  - ↓ [Annotation](#annotation-element)
+- ᴘ MaxLength: Int64 (optional)
+- ᴘ Name: String 
+- ᴘ Nullable: Boolean (optional)
+- ᴘ Precision: Int64 (optional)
+- ᴘ Scale: Scale (optional)
+- ᴘ SRID: Int64 (optional)
+- ʀ Type: [PrimitiveType](#primitivetype-element) | [ComplexType](#complextype-element) | [EntityType](#entitytype-element) | [EnumType](#enumtype-element) | [TypeDefinition](#typedefinition-element) 
+- ᴘ Unicode: Boolean (optional)
+
+##### <span id="primitivetype-element"/>PrimitiveType Element>/span>
+
+  - ↑ [Schema](#schema-element)
+  - ↓ [Annotation](#annotation-element)
+- ᴘ Abstract: Boolean 
+- ᴘ Name: String 
+
+##### <span id="property-element"/>Property Element>/span>
+
+  - ↑ [EntityType](#entitytype-element), [ComplexType](#complextype-element)
+  - ↓ [Annotation](#annotation-element)
+- ᴘ DefaultValue: String 
+- ᴘ MaxLength: Int64 (optional)
+- ᴘ Name: String 
+- ᴘ Nullable: Boolean (optional default: false))
+- ᴘ Precision: Int64 (optional)
+- ᴘ Scale: Scale (optional)
+- ᴘ SRID: Int64 (optional)
+- ʀ Type: [PrimitiveType](#primitivetype-element) | [ComplexType](#complextype-element) | [EnumType](#enumtype-element) | [TypeDefinition](#typedefinition-element) 
+- ᴘ Unicode: Boolean (optional)
+
+##### <span id="propertyref-element"/>PropertyRef Element>/span>
+
+  - ↑ [Key](#key-element)
+- ᴘ Alias: String (optional)
+- ᴄ Name: relative to ancestor 2 levels up
```

Comment:
Call out that the path must lead to a Property element (also elsewhere where ᴄ is used).